src/views/dataset/MultiSourceConfiguration.vue

```diff
@@ -696,6 +731,19 @@ export default class MultiSourceConfiguration extends Vue {
       channels = ["Default"];
     }
 
+    // Handle RGB expansion
+    if (this.isRGBFile && this.rgbBandCount > 1 && this.splitRGBBands) {
```

Comment:
I see this `this.isRGBFile && this.rgbBandCount > 1` a lot  
It might be worth putting it into a helper function